### DISCUSSION TITLE
Release page Update

### DISCUSSION THREAD

**Discussion by cb-rajeshkumar** [serenity-bdd/serenity-core#3036] [Q&A]
May I know why release page is not update after Feb 2022?

**Comment by wakaleo** [comment]
Because no one has updated it yet. Serenity BDD is an open source project - if you would like to help out, contributions are always welcome.

**Comment by globalworming** [comment]
somebody has to do it. I just did it but the release notes are a bit sparse :sweat_smile:

**Reply by Jacobvu84** [reply]
Maybe, I will do it when I have free time but you can teach me how to do it @globalworming
